server/portal/apps/datafiles/views.py

```diff
@@ -236,7 +236,6 @@ def put(self, request, filetype):
                 transfer_folder(src_client, dest_client, **body)
             else:
                 transfer(src_client, dest_client, **body)
-
```

Comment:
Is this deletion intentional?

Reply:
I don't think so. I'm not sure how that one happened since I was really only handling files in the `client` folder.
If you want to make the suggestion to add the space back in, I can fix that through here to correct it faster.